### DISCUSSION TITLE
How long before the operator will appear on the catalog?

### DISCUSSION THREAD

**Discussion by DanielePalaia** [k8s-operatorhub/community-operators#1529] [General]
Hi,

after a PR for a new operator get approved and merged how long before the operator will appear to the operatorhub catalog?

Thank you,
Daniele

**Comment by mvalarh** [comment]
After PR is merged. It should me within one hour or less. Depends if more operators were merged at same time. You can monitor process [here](https://github.com/k8s-operatorhub/community-operators/actions/workflows/operator_release.yaml).
